Jenkinsfile

```diff
@@ -268,6 +383,9 @@ pipeline {
             always {
               junit testResults: '**/target/*-reports/TEST-*.xml', keepLongStdio: true
             }
+            failure {
+              addFailedStageType(failedStageTypes, 'webapp-IT')
+            }
```

Comment:
Tomcat IT doesn't have downstream stages
```suggestion
```

Reply:
Removed this block in two places `engine-IT-tomcat-postgresql-96` and `webapp-IT-tomcat-h2`.